SPECS/interface-rename.spec

```diff
@@ -1,13 +1,16 @@
-%global package_speccommit a621c540eb018b28c20bc5d16004028e24d8d2d1
-%global package_srccommit v2.0.5
+%global package_speccommit a641d796e17072781218dbc1ec8f1af1064e3180
+%global package_srccommit v2.0.6
 Summary:        A program that rename network interfaces to keep them consistent
 Name:           interface-rename
-Version: 2.0.5
-Release:        1.1%{?xsrel}%{?dist}
+Version: 2.0.6
+Release:        1%{?xsrel}.1%{?dist}
 License:        GPLv2+
 Group:          System Environment/Base
-Source0: interface-rename-2.0.5.tar.gz
-Patch0:  0001-More-py3-fixes.patch
+Source0: interface-rename-2.0.6.tar.gz
+
+# XCP-ng Patches
+# TODO Drop it?
+Patch1000:  0001-More-py3-fixes.patch
```

Comment:
his is right, this patch has been applied upstream, it does not apply any more

Reply:
yes I just checked and we can drop it.